client/dive-common/components/TrackDetailsPanel.vue

```diff
@@ -495,6 +511,54 @@ export default defineComponent({
           <v-spacer />
           Abort (esc)
         </v-btn>
+        <v-btn
+          v-if="editingMultiTrack"
+          color="error"
+          class="mx-2 mb-2 grow"
+          :disabled="readOnlyMode"
+          depressed
+          x-small
+          @click="$emit('delete-selected-tracks')"
+        >
+          <v-icon
+            class="pr-1"
+            small
+          >
+            mdi-delete
+          </v-icon>
+          <v-spacer />
+          Delete selected tracks
+        </v-btn>
+        <div
+          v-if="editingMultiTrack"
+          class="d-flex justify-center align-center mb-2 mx-2"
+          width="100%"
+        >
+          <v-spacer />
+          <v-label class="mx-2">
+            Type:
+          </v-label>
+          <TypePicker
+            :value="multiTrackType"
+            :all-types="allGroupTypesRef"
+            :read-only-mode="readOnlyMode"
+            selected
+            update-on-input
+            data-list-source="allGroupTypesOptions"
+            @input="updateMultiTrackType"
+          />
```

Comment:
I think we should add the `lock-types` prop to this component from `clientSettings`.  This is a functionaltiy that if in the Type List settings you can prevent users from entering in their own custom types.  It changes the behavior of the TypePicker slightly so it is dropdown instead of a combo box and limits what types can be used.

Reply:
d5d4e3ab imports `clientSettings` and uses it for this component.